.env.example

```diff
@@ -13,11 +13,11 @@
 # https://openid.net/specs/openid-connect-discovery-1_0.html#ProviderConfigurationRequest
 # In Keycloak, this is realm-local; do not include the trailing
 # `/.well-known/openid-configuration` path
-REACT_APP_OIDC_AUTHORITY=https://auth.example.com/realms/example
+VITE_REACT_APP_OIDC_AUTHORITY=https://auth.example.com/realms/example
```

Comment:
Realizing that we should remove `REACT_APP` from all of these env vars.

So instead of `VITE_REACT_APP_FOO` it would just be `VITE_FOO`.